doors_dashboards/dashboards/dashboard.py

```diff
@@ -87,6 +105,11 @@ def create_dashboard(config: Dict) -> Dash:
         id=dashboard_id,
         fluid=True,
         children=[
+            dcc.Store(id='map-value-store', storage_type='local'),
+            dcc.Store(id='general'),
+            dcc.Store(id='collection_selector'),
+            dcc.Store(id='group_selector'),
+            dcc.Store(id="variable_selector"),
```

Comment:
We need to checkt whether stores need to be declared at this level or whether they can be declared within the components that require them,

Reply:
I think it depends on us how we want to make the changes,  if we want it from the  beginning of the page load then we need to declare it in dashboard else we declare dynamically when needed.

Reply:
I don't think a dynamic declaration is possible. I was just wondering about the visibility scope. If a store is only used by, e.g., the collection selection, it would be prefereable if it were only there.

Reply:
Ok we can discuss.